test/integration/007_graph_selection_tests/test_graph_selection.py

```diff
@@ -24,6 +24,36 @@ def test__postgres__specific_model(self):
         self.assertFalse('base_users' in created_models)
         self.assertFalse('emails' in created_models)
 
+    @attr(type='postgres')
+    def test__postgres__tags(self):
+        self.use_profile('postgres')
+        self.use_default_project()
```

Comment:
You don't need `use_default_project` or `use_profile('postgres')` in tests anymore unless you're changing project/profile data inside the test, `setUp` calls them.

Reply:
these should be using `@use_profile('postgres')` though, right? I missed that the first time around

Reply:
`use_profile` vs `attr` is redundant now, profile/project things happen in `setUp` based on the method name.